roles/openshift_hosted/templates/registry_config.j2

```diff
@@ -66,13 +66,13 @@ middleware:
   - name: openshift
     options:
       pullthrough: {{ openshift.hosted.registry.pullthrough | default(true) }}
-{% if openshift.hosted.registry.storage.provider == 's3' and 'cloudfront' in openshift.hosted.registry.storage.s3 %}
+{% if openshift.hosted.registry.storage.provider == 's3' and openshift_hosted_registry_storage_s3_cloudfront is defined %}
```

Comment:
`openshift_hosted_registry_storage_s3_cloudfront` will never be defined on its own so we should probably check for `openshift_hosted_registry_storage_s3_cloudfront_baseurl` as an alternative for now.

We can remove the fact for `openshift.hosted.registry.storage.provider` as long as we define a default for the comparison.

```
{% if openshift_hosted_registry_storage_provider | default('') == 's3' and openshift_hosted_registry_storage_s3_cloudfront_baseurl is defined %}
```